addons/im_livechat/static/src/core/web/command_palette.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.LivechatChannelCommand">
+        <div class="o_command_default d-flex align-items-center justify-content-between px-4 py-2">
+            <i class="me-3" t-att-class="{
+                'fa fa-sign-out text-danger': props.channel.hasSelfAsMember,
+                'fa fa-sign-in text-success': !props.channel.hasSelfAsMember
+            }"/>
```

Comment:
the icon being confusing, I would personally find it more clear if the command also displayed the action text (join/leave)

Reply:
Added